projects/packages/sync/src/modules/class-module.php

```diff
@@ -59,6 +59,29 @@ public function table_name() {
 		return false;
 	}
 
+	/**
+	 * The table in the database. This can be overridden to allow tables not present in the $wpdb global.
+	 *
+	 * @access public
+	 *
+	 * @return string|bool
+	 */
+	public function table() {
```

Comment:
Created a new method so that `table_name` is still available and the changes in the existing code are minimal. Maybe we could repurpose the existing `table_name` one instead.

I did some auditing the way the changes are now and couldn't find any other places needing modifications.

Reply:
Nice thanks! Just repeating here what we already discussed to raise awareness:
Since we find it confusing to keep both `table` and `table_name` methods, we agreed that we are going to introduce the `table` method, which will return the table via either `$wpdb`  or with the prefix for custom tables that are not available via `$wpdb`.
This means that consumers of this method, like `Module` will not have to use `$wpdb` anymore.
We'll make the new method completely independent from `table_name()` so that we can deprecate the later and remove it in a future release.